comixed-model/src/main/java/org/comixedproject/model/comicbooks/Comic.java

```diff
@@ -368,6 +368,8 @@
   private Date lastModifiedOn = new Date();
 
   @OneToOne(mappedBy = "comic", cascade = CascadeType.ALL, orphanRemoval = true)
+  @JsonProperty("metadataSource")
```

Comment:
I wonder if the name of this property might lead to confusion. Wouldn't it be better to name it comicMetadataSource to differentiate it from the MetadataSource class?